pkg/cmd/roachtest/test_runner.go

```diff
@@ -2036,12 +2036,13 @@ func (r *testRunner) postProcessPerfMetrics(
 }
 
 func (m *perfMetricsCollector) collectFromNodes(
-	c *clusterImpl, dstDirFn func(nodeIdx int) string,
+	c *clusterImpl, dstDirFn func(nodeIdx int) string, log *logger.Logger,
 ) error {
 	for _, node := range getPerfArtifactsNode(c) {
 		files, err := m.findMetricsFiles(dstDirFn(node))
 		if err != nil {
-			return errors.Wrapf(err, "finding metrics files")
+			log.Printf("[warning] failed to find metrics files, will continue: %s", err)
+			continue
 		}
 
 		if err := m.processFiles(files); err != nil {
```

Comment:
Should we throw an error if we never successfully call `processFiles` on any of the nodes?